config/environments/production.rb

```diff
@@ -92,4 +92,12 @@
   # ]
   # Skip DNS rebinding protection for the default health check endpoint.
   # config.host_authorization = { exclude: ->(request) { request.path == "/up" } }
+
+  def configure_oidc
```

Comment:
maybe this should be a default implementation and then dev and test just set issuer, client_id, and client_secret config first?

Reply:
done